packages/@aws-cdk/aws-codedeploy/lib/server/application.ts

```diff
@@ -58,53 +34,74 @@ export interface ServerApplicationProps {
    *
    * @default an auto-generated name will be used
    */
-  applicationName?: string;
+  readonly applicationName?: string;
 }
 
 /**
  * A CodeDeploy Application that deploys to EC2/on-premise instances.
  */
 export class ServerApplication extends cdk.Construct implements IServerApplication {
-  /**
-   * Import an Application defined either outside the CDK,
-   * or in a different CDK Stack and exported using the {@link #export} method.
-   *
-   * @param parent the parent Construct for this new Construct
-   * @param id the logical ID of this new Construct
-   * @param props the properties of the referenced Application
-   * @returns a Construct representing a reference to an existing Application
-   */
-  public static import(scope: cdk.Construct, id: string, props: ServerApplicationImportProps): IServerApplication {
+  public static import(scope: cdk.Construct, id: string, props: ServerApplicationImportProps) {
```

Comment:
Why remove the docs, and return types...?

Reply:
Was messing around with the interfaces, things got lost during the refactor.